content/draft/general/keyboard_shortcuts.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Keyboard shortcuts spec"
+description: ""
+date: 2020-12-01T08:48:57+00:00
+lastmod: 2020-12-01T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "general"
+weight: 100
+toc: true
+---
+
+# Keyboard Shortcuts spec
+
+## Motivation
+
+Users often use keyboard shortcuts to more effectively navigate applications. This spec aims to document the keyboard shortcuts available as well as any preconditions.
+
+## Designs
+
+* [designs](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=355%3A82493)
+
+## Functional Requirements
+
+### Key definition
+
+The requirements will use the keys `CMD` and `OPT` which are defined for each platform as follows:
+
+- `CMD`
+    - macOS: `⌘` or `CMD`
+    - Windows: `Ctrl`
+    - Linux: `Ctrl`
+- `OPT`
+    - macOS: `⌥` or `OPT`
+    - Windows: `Alt`
+    - Linux: `Alt`
+
+### Keyboard shortcuts help
+
+- `CMD + /` MUST open a modal listing available key shortcuts
+- The Modal listing available keyshortcuts MUST display the correct keys depending on the platform, for example `CMD + C` on macOS and `Ctrl + C` on Windows.
+
+### Keyboard shortcuts
+
+#### Message focus
+
+- `TAB` - MUST trigger focus of the last message
```

Comment:
we're currently missing the design for a 'focused message' (or at least I can't find it)

Reply:
found a design here https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5395%3A46518

@John-44 can you confirm please if this is still the intended design for focusing on a message with keyboard shortcuts?

Reply:
@iurimatias no that figma link is not the intended design, that figma is out of date.  Kuba is working updating the focus state designs as his next task, hopefully these designs will be ready some time next week.

Reply:
Focus state designs are here https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1759%3A332242 but I do think spatial focus based interaction should be a separate spec.